go/oasis-net-runner/fixtures/fixtures.go

```diff
@@ -0,0 +1,70 @@
+// Package fixtures provides network configuration fixtures.
+package fixtures
+
+import (
+	"encoding/json"
+	"math"
+
+	flag "github.com/spf13/pflag"
+	"github.com/spf13/viper"
+
+	"github.com/oasislabs/oasis-core/go/common"
+	"github.com/oasislabs/oasis-core/go/oasis-test-runner/oasis"
+)
+
+const (
+	cfgFile = "fixture.file"
+)
+
+var (
+	// Flags are command line flags for the fixtures.
+	Flags = flag.NewFlagSet("", flag.ContinueOnError)
+
+	// DefaultFlags are  command line flags for the fixture.default.* flags.
+	DefaultFlags = flag.NewFlagSet("", flag.ContinueOnError)
+
+	runtimeID    common.Namespace
+	keymanagerID common.Namespace
+)
+
+// GetFixture generates a fixture object from given file or default fixture, if no fixtures file provided.
+func GetFixture() (f *oasis.NetworkFixture, err error) {
+	if viper.IsSet(cfgFile) {
+		f, err = newFixtureFromFile(viper.GetString(cfgFile))
+	} else {
+		f, err = newDefaultFixture()
+	}
+	if err != nil {
+		return
+	}
+
+	return
+}
+
+// DumpFixture dumps given fixture to JSON-encoded bytes.
+func DumpFixture(f *oasis.NetworkFixture) ([]byte, error) {
+	data, err := json.Marshal(f)
+	if err != nil {
+		return nil, err
+	}
+
+	return data, nil
+}
+
+func init() {
+	DefaultFlags.Bool(cfgEpochtimeMock, false, "use mock epochtime")
```

Comment:
Since you moved the constants also move this part of the init function, together with runtimeID and keymanagerID, to `default.go`.